docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -10,113 +10,155 @@ You can push {es-sec} cases to these third-party systems:
-you are sending cases.
-* *Email address* ({jira} connectors only): The {jira} account's username or email address.
-* *API token* ({jira} connectors only): The API token or password used
-to authenticate {jira} updates.
-* *API key ID* ({ibm-r} connectors only): The API key used to authenticate
-{ibm-r} updates.
-* *API key secret* ({ibm-r} connectors only): The API key secret used to
-authenticate {ibm-r} updates.
-* *API token* ({swimlane} connectors only): The {swimlane} API authentication token used for HTTP Basic authentication. This is the personal access token for your user role.
-
-. Choose the connector type (for {swimlane} connectors only):
-** *All*: You can choose to set all or no field mappings when creating your new {swimlane} connector. However, note that if you don't set field mappings now, you'll be prompted to do so if you want to use the connector for a case or a rule. The prompts no longer display once you set up the required mappings.
-** *Alerts*: Provide an alert ID and rule name.
-** *Cases*: Provide a case ID, a case name, comments, and a description.
+IMPORTANT: If you've upgraded from {stack} version 7.15.0 or earlier to version 7.16.0 or later, you must install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] or https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] on your {sn} instance and complete additional configuration steps before creating a new {sn} ITSM or SecOps connector. For more information, refer to {kibana-ref}/servicenow-action-type.html[{sn} ITSM] and {kibana-ref}/servicenow-sir-action-type.html[{sn} SecOps].
+
+. Enter your required settings.
++
+|===
+
+| *Connector name* | Name for the connector.
+
+| *URL* | The URL of the external system to which you want to send cases.
+
+| *{sn} instance URL* | ({sn} only) The URL of the {sn} instance to which you want to send cases.
+
+| *Use OAuth authentication* | ({sn} only) Enable the toggle to authenticate a connection between Elastic and {sn}. Refer to the {sn} {kibana-ref}/action-types.html[Connector] topics for more information.
+
+| *API URL* |  ({swimlane} only) The URL of the {swimlane} instance to which you want to send cases.
+
+| *Organization ID* | ({ibm-r} only) Your organization’s {ibm-r} ID number.
+
+| *Application ID* | ({swimlane} only) The application ID of your {swimlane} application. From {swimlane}, you can find the application
+ID by checking your application’s settings or at the end of your application’s URL after you’ve opened it.
+
+| *Username* | ({sn} only) The username of the {sn} account used to access the {sn} instance.
+
+| *Password* | ({sn} only) The password of the {sn} account is used to access the {sn} instance.
 
+| *Project key* | ({jira} only) The key of the {jira} project to which you are sending cases.
```

Comment:
There are a bunch of fields that are required for ServiceNow if Use OAuth is true: Client ID, Client Secret, JWT Key ID, Private Key, Private Key Password. Should these all be added?

Reply:
Yep! I'll refer to the ServiceNow stack management docs for field definitions.